Video.js

```diff
@@ -180,7 +180,7 @@ export default class Video extends Component {
         uri,
         isNetwork,
         isAsset,
-        type: source.type,
+        type: source.type || 'mp4',
```

Comment:
It looks removed by #426, but it will broken native prop update (if it is `undefined`), so I added back it.